frontend/src/app/modules/admin-area/requests/requests-modal.component/requests-modal.component.ts

```diff
@@ -49,7 +49,11 @@ export class RequestsModalComponent implements OnInit {
   }
 
   closeForm() {
-    this.activeModal.close();
+    this.activeModal.close('Send');
+  }
+
+  cancelForm() {
+    this.activeModal.close('Cancel');
```

Comment:
you can dismiss the modal this.activeModal.dismiss)

Reply:
Okay, I corrected it.

Reply:
But it will throw you an error, so you should catch it